guides/advanced_guides/configuration.md

```diff
@@ -287,39 +295,43 @@ This command will stop the process if:
 
 If this command is not called, no snapshot will be imported.
 
-### Ignore missing snapshot
+### Ignore Missing Snapshot
 
 **Environment variable**: `MEILI_IGNORE_MISSING_SNAPSHOT`
 **CLI option**: `--ignore-missing-snapshot`
 
 The engine ignores missing snapshots and does not throw an error in this case.
```

Comment:
```suggestion
The engine ignores missing snapshots in the case of a `--snapshot-import`, and does not throw an error in this case.
```

Reply:
```suggestion
The engine ignores missing snapshots in the case of a `--import-snapshot`, and does not throw an error in this case.
```

Reply:
I want to make it clear that this variable has to do with the import snapshot process.

Reply:
just show you the real option, because it's less natural in reading with this version but "more true" in the usage

Reply:
Ah yes! Ok I had to read it 200 times before seeing the difference. My bad!